packages/components/package.json

```diff
@@ -22,6 +22,7 @@
         "@datastax/astra-db-ts": "^0.1.2",
         "@dqbd/tiktoken": "^1.0.7",
         "@elastic/elasticsearch": "^8.9.0",
+        "@getzep/zep-cloud": "npm:@getzep/zep-js@next",
```

Comment:
Note: We are aliasing a different version of our ts sdk (tagged with `next`) as we currently only have support for Zep Cloud API in Release Candidate of our ts sdk